src/intervals/set_operations.jl

```diff
@@ -119,6 +119,8 @@ hull(a::Interval, b::Interval) = Interval(min(a.lo, b.lo), max(a.hi, b.hi))
 # hull{T,S}(a::Interval{T}, b::Interval{S}) = hull(promote(a, b)...)
 hull(a::Complex{<:Interval},b::Complex{<:Interval}) =
     complex(hull(real(a),real(b)),hull(imag(a),imag(b)))
+hull(a...) = reduce(hull, a)
+hull(a::Vector{Interval{T}}) where {T}= hull(a...)
```

Comment:
You can just use reduce directly here I think, rather than splatting the vector.

Reply:
Good catch! I guess I simply forgot this.